client/src/containers/ReviewCode.js

```diff
@@ -1,37 +1,103 @@
 import React, { PropTypes, Component } from 'react';
-import { fetchCode } from '../actions/code';
+import { fetchCode, selectCode } from '../actions/code.js';
+import { addCodeSmells } from '../actions/smells';
+import { getUserInfo } from '../actions/user.js';
 import { connect } from 'react-redux';
+import CodeBlock from '../components/CodeBlock';
+
 
 class ReviewCode extends Component {
     constructor(props) {
         super(props);
+        this.clickAction = this.clickAction.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.selectCodeSmell = this.selectCodeSmell.bind(this);
+        this.state = {
+            codeSmellName: ""
+        }
     }
 
     componentDidMount() {
-        const { dispatch, id, codeReview } = this.props;
+        const { dispatch, id } = this.props;
         dispatch(fetchCode(id));
+        dispatch(getUserInfo());
```

Comment:
Yes, otherwise, the state.user info defaults to John Doe. Do you have a better implementation suggestion?